how-to/README.md

```diff
@@ -1,12 +1,22 @@
-# Howto guides
+# Howto Guides
 
+* [Howto Guides](#howto-guides)
```

Comment:
Nice split out - thanks! James and I have always wondered why we have both howto and usecases - and have wondered if they should all go in one place ;-)